R/TuningInstanceSingleCrit.R

```diff
@@ -115,18 +115,26 @@ TuningInstanceSingleCrit = R6Class("TuningInstanceSingleCrit",
     #' This defines the resampled performance of a learner on a task, a
     #' feasibility region for the parameters the tuner is supposed to optimize,
     #' and a termination criterion.
-    initialize = function(task, learner, resampling, measure, search_space,
-      terminator, store_benchmark_result = TRUE, store_models = FALSE,
-      check_values = FALSE) {
-        measure = as_measure(measure)
-        obj = ObjectiveTuning$new(task = task, learner = learner,
-          resampling = resampling, measures = list(measure),
-          store_benchmark_result = store_benchmark_result,
-          store_models = store_models, check_values = check_values)
-        super$initialize(obj, search_space, terminator)
-        self$archive = ArchiveTuning$new(search_space = search_space,
-          codomain = self$objective$codomain, check_values = check_values)
-        self$objective$archive = self$archive
+    initialize = function(task, learner, resampling, measure,
+      search_space = NULL, terminator, store_benchmark_result = TRUE,
```

Comment:
Now that the search space is optional it would make sense to put it after the terminator? Does that break a lot of stuff?

Reply:
I changed it.